Base/usr/share/man/man1/sleep.md

```diff
@@ -0,0 +1,23 @@
+## Name
+
+sleep - delay for a specified amount of seconds
```

Comment:
[Amount is used for uncountable nouns. Let's maybe use number instead.](https://dictionary.cambridge.org/grammar/british-grammar/amount-of-number-of-or-quantity-of)

Reply:
Sure!